### ISSUE TITLE
Input fields almost unusable

### ISSUE THREAD

**Issue by rgpublic** [ctf0/Seti_ST3#310]
I don't know when it started - perhaps only with the most recent ST4 update - but text in all input fields is almost invisible:

![grafik](https://user-images.githubusercontent.com/17726950/132379951-981c0e03-2a69-4b35-bd06-65b1b12cacb4.png)

The whole input field has a white background color. As soon as I select some text, the whole input field gets the normal dark background again.

**Comment by tuzemec** [comment]
Same for me. I think it started with the latest update (4114) in the Dev build channel.

**Comment by ctf0** [comment]
i stopped using st, so any PRs are welcome.

**Comment by rgpublic** [comment]
Ah, good to know that someone else experiences this and I'm not just seeing things. I panicked a bit, because I thought I had irrevocably misconfigured my machine ot sth.

I found the following emergency workaround. It's a few minutes of work and a little bit complicated, but since Sublime is almost unusable without it, perhaps others find this helpful until a true patch if found:

* Preferences => Browser packages (Sublime's "Packages" folder is opened in the file manager)
* Change to "User" directory
* Create a folder "Seti_UI"
* Inside, create another folder "Widgets"
* Copy the file "Widget - Seti.stTheme" over from Packages/Seti_UI/Widgets (Same path but without "User")
* Open the XML file under "User" and change the "foreground" key to "#000000". Save.
* Basically what you did up until here is clone a single file from the original package in your Users folder and adapt it.
* Now, back in the original package folder (not the "Users" one), open Widget - Seti.sublime-settings or Widget - Seti_orig.sublime-settings, respecitvely - depending on which theme you use.
* Under "color_scheme" change the path to your new folder by inserting "/User". The value should now be: "Packages/User/Seti_UI/Widgets/Widget - Seti.stTheme"
